include/utility/json_utility.hpp

```diff
@@ -1003,5 +1003,46 @@ inline bool isFruReplaceableAtStandby(const nlohmann::json& i_sysCfgJsonObj,
 
     return false;
 }
+
+/**
+ * @brief API to get list of FRUs replaceable at standby.
+ *
+ * @param[in] i_sysCfgJsonObj - System config JSON object.
+ *
+ * @return - List of FRUs replaceable at standby.
```

Comment:
I am ok not having a tray..catch in this block as long as the oxygen tells that it could throw , so that it will be the owners responsibility to catch the exception and handle it

Reply:
Done.